lumigator/python/mzai/backend/backend/alembic/versions/ef5ee5662ce3_add_job_type_to_job_table.py

```diff
@@ -0,0 +1,26 @@
+"""Add job type to job table
+
+Revision ID: ef5ee5662ce3
+Revises: e9679cbc3c36
+Create Date: 2025-01-07 13:24:40.657958
+
+"""
+
+from collections.abc import Sequence
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "ef5ee5662ce3"  # pragma: allowlist secret
+down_revision: str | None = "e9679cbc3c36"  # pragma: allowlist secret
+branch_labels: str | Sequence[str] | None = None
+depends_on: str | Sequence[str] | None = None
```

Comment:
Let's make sure when we merge this it doesn't conflict with other alembic changes

Reply:
Hmmmmm, true :-/ I'll regenerate the alembic part just before merging just in case.